reducers/index.ts

```diff
@@ -0,0 +1,35 @@
+import { ActionType, Action } from "../actions";
+import { StateInterface } from "../context/state";
+
+export const gameReducer = (state: StateInterface, action: Action) => {
```

Comment:
instead of using Redux to manage state we're using a Context and this 'useReducer' react function that allows you return a new state with each action. we'll eventually add specific state updates to each action here that will update the values in the StateInterface (like score, level, etc)